tests/playbooks/tests_ethtool_features.yml

```diff
@@ -155,12 +161,17 @@
             - name: Show network_connections result
               debug:
                 var: __network_connections_result
-            - assert:
+            - name: Assert the duplicate key "tx_tcp_segmentation"
+                configuration error happened
+              vars:
+                errmsg: >-
+                  fatal error: configuration error:
+                  connections[0].ethtool.features:
+                  duplicate key 'tx_tcp_segmentation'
+              assert:
                 that:
-                  - '{{  "fatal error: configuration error:
-                      connections[0].ethtool.features: duplicate key
-                      ''tx_tcp_segmentation''" in
-                      __network_connections_result.msg }}'
+                  - errmsg in __network_connections_result.msg
```

Comment:
This change is not related to the naming issue

Reply:
I updated the patch, thanks.